praw/const.py

```diff
@@ -91,14 +91,17 @@
     'my_subreddits':          'subreddits/mine/subscriber/',
     'marknsfw':               'api/marknsfw/',
     'modmail_archive':        'api/mod/conversations/{id}/archive',
+    'modmail_bulk_read':      'api/mod/conversations/bulk/read',
```

Comment:
N.b. This endpoint is different from what's documented at https://www.reddit.com/dev/api/.